docker/automatic-dev-environment-setup.sh

```diff
@@ -0,0 +1,22 @@
+#if mac:
+# check if git is installed on mac
+echo "🥫 Welcome to the Open Food Facts dev environment setup"
+echo "🥫 Checking git is installed…"
+git --version
+echo "🥫 Cloning Open Food Facts web server - product-opener…"
+git clone https://github.com/openfoodfacts/openfoodfacts-server.git
+echo "🥫 Installing Docker… you may have to open it manually to grant permissions"
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+brew cask install docker
```

Comment:
not everyone uses `brew cask`, I prefer the Desktop install of Docker on Mac for instance.